### DISCUSSION TITLE
Kraken 2023 Model Not Listed (240 Cooler)

### DISCUSSION THREAD

**Discussion by Srhandsome** [liquidctl/liquidctl#654] [Q&A]
I'm still pretty new to Linux (Ubuntu 22.04.3) and have tried to do everything I could before reaching out for help. 
The guide is extensive and wonderful at addressing potential issues. Mine seems to be that 
`lsusb`
recognizes my 
`Bus 001 Device 005: ID 1e71:300e NZXT NZXT Kraken Base`
but only device I don't get it when checking in `liquidctl`

> $ liquidctl list
Device ID 0: Gigabyte RGB Fusion 2.0 5702 Controller

My issue is that when I have attempted to unbind from HID using the troubleshoot guide but still get an error

>~$ echo '1-7:1.1' | sudo tee /sys/bus/usb/drivers/usbhid/bind
1-7:1.1
tee: /sys/bus/usb/drivers/usbhid/bind: Device or resource busy

**Comment by aleksamagicka** [comment]
Support for that Kraken in liquidctl has been merged in [October](https://github.com/liquidctl/liquidctl/commit/846ac5f5aa9d5484e8ed1d549fd2478bbf51fdd8). Please retry with the [Git version](https://github.com/liquidctl/liquidctl#installing-from-pypi-or-github) for the time being.

**Reply by Srhandsome** [reply]
You are a legend. Thank you
